PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/PostalRateCalculatorApplicationTest.java

```diff
@@ -7,6 +7,6 @@
 public class PostalRateCalculatorApplicationTest {
 
     @Test
-    public void contextLoads() {}
+    private void contextLoads() {}
```

Comment:
why

Reply:
comme ca it doesnt run en meme tps que les autres . That way we have the same number of tests as documented entries